helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -349,6 +349,30 @@ spec:
         service: cilium-agent
       record: slo_target
 
+    # -- VPA
+    # Amount of request for VPA
+    - expr: "count(up{app=~'vertical-pod-autoscaler.*'}) by (cluster_type,cluster_id)"
+      labels:
+        class: MEDIUM
+        area: platform
+        service: vertical-pod-autoscaler
+      record: raw_slo_requests
+
+    # Amout of errors for VPA
+    - expr: "sum((up{app=~'vertical-pod-autoscaler.*'} * -1) + 1) by (cluster_id, cluster_type)"
```

Comment:
this is not a trivial query (due to the `* -1 +1` part) and it's not self explanatory. Can you please add a comment to explain how it works?